share/spice/mass_on_time/mass_on_time.js

```diff
@@ -57,7 +108,9 @@ function ddg_spice_mass_on_time (api_result) {
     Spice.add({
```

Comment:
It's undocumented at the moment, but please wrap this in `DDG.require` because, when deployed, we need to actually load mapbox.js:

```javascript
DDG.require('maps', function() {
    Spice.add(...);
});
```

Thanks!